app.js

```diff
@@ -21,7 +21,9 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
-app.use('/', userUUIDMiddleware, require('./routes/index'));
+app.use(userUUIDMiddleware);
+
+app.use('/', require('./routes/index'));
```

Comment:
Because it will fall through to the `/interface` and `/voice` endpoints, as they match the pattern. Specifying it on the specific endpoints prevents that from happening.